tests/unit/App/DataSource/DataSourceStorageTest.php

```diff
@@ -105,31 +140,34 @@ public function testFailedReceive()
             ])
             ->andReturn($this->usecase);
 
-        $e = M::spy(\Ushahidi\Core\Exception\NotFoundException::class);
+        $e = M::mock($thrownE);
+        $e->allows()->getErrors()->andReturns(["sorry my fault", "but you are also not perfect"]);
 
         $this->usecase
             ->shouldReceive('interact')
             ->once()
             ->andThrow($e);
 
-        $storage->receive(
-            'smssync',
-            'sms',
-            'phone',
-            123456,
-            'Yo dawg I heard you like messages, so I put some messages in your messages',
-            "YOU!",
-            null,
-            null,
-            null,
-            null,
-            1,
-            ['Title' => 'somekey']
-        );
-
-        $e->shouldHaveReceived('getMessage')->once();
```

Comment:
that line was never ran and even if it would have been it would not worked, because `getMessage` cannot be mocked see https://github.com/mockery/mockery/issues/581